pvlib/irradiance.py

```diff
@@ -16,22 +16,36 @@
 from pvlib import atmosphere, solarposition, tools
 import pvlib  # used to avoid dni name collision in complete_irradiance
 
+from pvlib._deprecation import pvlibDeprecationWarning
+import warnings
+
+
+# Deprecation warning based on https://peps.python.org/pep-0562/
+def __getattr__(attr):
+    if attr in ['SURFACE_ALBEDOS']:
+        warnings.warn(f"{attr} has been moved to the albedo module as of "
+                      "v0.11.0. Please use pvlib.albedo.SURFACE_ALBEDOS.",
+                      pvlibDeprecationWarning)
+        return globals()[f"_DEPRECATED_{attr}"]
+    raise AttributeError(f"module {__name__!r} has no attribute {attr!r}")
+
 
 # see References section of get_ground_diffuse function
-SURFACE_ALBEDOS = {'urban': 0.18,
-                   'grass': 0.20,
-                   'fresh grass': 0.26,
-                   'soil': 0.17,
-                   'sand': 0.40,
-                   'snow': 0.65,
-                   'fresh snow': 0.75,
-                   'asphalt': 0.12,
-                   'concrete': 0.30,
-                   'aluminum': 0.85,
-                   'copper': 0.74,
-                   'fresh steel': 0.35,
-                   'dirty steel': 0.08,
-                   'sea': 0.06}
+_DEPRECATED_SURFACE_ALBEDOS = {
```

Comment:
We should not rename the variable here (otherwise it is a breaking change, not merely a deprecation), but I think a better approach for here in `irradiance.py` is to use `from pvlib.albedo import SURFACE_ALBEDOS` so that we don't have redundant dictionary definitions.